### DISCUSSION TITLE
Discussão sobre a atividade de internacionalização

### DISCUSSION THREAD

**Discussion by luana-martins** [mate28-ic-ufba/turma-20212#114] [Q&A]
Olá pessoal :smiley: 

Uma curiosidade, qual o feedback que vocês receberam na atividade de internacionalização?

A minha contribuição no SpotBugs foi bem recebida. O contribuidor mais ativo nas documentações me deu todo suporte e criou um  guia para que outras pessoas contribuam para a internacionalização do projeto.

Meu [PR](https://github.com/spotbugs/spotbugs/pull/1796) foi revisado em relação aos padrões de documentação, mas não a tradução em si. O revisor disse que vai tentar encontrar outros para verificar a tradução. Até onde sei, sou só eu de contribuidora BR no projeto. 

Estava pensando que vocês obtiveram um feedback parecido, talvez possamos revisar as traduções uns dos outros. Acho que seria uma forma de fornecer algo mais confiável para o projeto, sabe? 
Me digam aí, o que vocês acham?

**Comment by levysantiago** [comment]
Eu acho a ideia super bacana, seria um crossover dos projetos :). Vamos ver se tiveram feedbacks parecidos e o que a @danielafeitosa e a @christinaflach acham desta ideia também.

**Comment by danielafeitosa** [comment]
@luana-martins eu acho que é uma ótima ideia! Como vc é a única que sabe falar português no projeto, eles vão confiar que sua tradução tá certa até encontrarem alguém. Seria muito bom outra pessoa que fala português validar sua tradução!
E isso é bem comum em projetos que estão iniciando a internacionalização.
